### DISCUSSION TITLE
Fix spacing of arrow functions

### DISCUSSION THREAD

**Discussion by windaishi** [PHP-CS-Fixer/PHP-CS-Fixer#7612] [Q&A] [answered]
Hello,

I am looking for a fixer that will fix the following white space error in closures:

```
fn(array $error) =>  new JsonApiError($error);
```
to 
```
fn(array $error) => new JsonApiError($error);
```

(Remove the double space after `=>`)

Is there already any rule that can fix this?

Thank you for your help

**Comment by Wirone** [comment, marked as answer]
You need `binary_operator_spaces`, please take a look [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/8058b058bedca7af86f59ab082bde4257c9394e3/tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php#L3289) 🙂.
